charts/identity-trust-bundle/README.md

```diff
@@ -0,0 +1,120 @@
+# Identity & Trust Bundle
+
+The Identity & Trust Bundle provides identity management and authentication services for the Catena-X ecosystem. It includes the Identity Wallet Mock and OAuth2 Authorization Server.
+
+## Components
+
+- **Identity Wallet Mock**: Simulates identity wallet functionality for testing and development
+- **OAuth2 Authorization Server**: Provides OAuth2 and OpenID Connect authentication services
+- **Identity Provider Services**: Manages user identities and authentication
+
+## Prerequisites
+
+- Kubernetes cluster (version 1.24 or higher)
+- Helm (version 3.8 or higher)
+- kubectl configured to access your cluster
+
+## Installation
+
+### Standalone Installation
+
+To install the Identity & Trust Bundle standalone:
+
+```bash
+# Add the repository
+helm repo add tractusx https://eclipse-tractusx.github.io/charts/dev
+
+# Install the bundle
+helm install identity-trust-bundle ./charts/identity-trust-bundle
+```
+
+### Installation with Umbrella Chart
+
+To install as part of the umbrella chart:
+
+```bash
+helm install umbrella ./charts/umbrella --set identity-trust-bundle.enabled=true
+```
+
+## Configuration
+
+The bundle can be configured through the `values.yaml` file or using `--set` flags. Key configuration options include:
+
+### Identity Wallet Mock Configuration
+
+```yaml
+identity-wallet-mock:
+  enabled: true
+  image:
+    repository: eclipse-tractusx/identity-wallet-mock
+    tag: "1.0.0"
+  service:
+    type: ClusterIP
+    port: 8080
+```
+
+### OAuth2 Server Configuration
+
+```yaml
+oauth2-server:
+  enabled: true
+  config:
+    oauth2:
+      clients:
+        - clientId: "your-client-id"
+          clientSecret: "your-client-secret"
+          redirectUris:
+            - "http://your-app:8080/login/oauth2/code/oauth2"
```

Comment:
```suggestion
            - "http://<your-app>:8080/login/oauth2/code/oauth2"
```

Reply:
Pls use common variables in documentation like in the following sections /<service-name>...